source/docs/hardware-reference/talonfx/improving-performance-with-current-limits.rst

```diff
@@ -0,0 +1,215 @@
+Improving Performance with Current Limits
+=========================================
+
+Current-limiting, is the process of reducing and eliminating motor output when a given current has surpassed a limit. There are 3 types of current limits available: supply, stator, and torque. Each of these limits accomplishes different goals. This article goes over how to configure those limits, when to configure them and why current limiting is important.
```

Comment:
"Current limiting is the process of restricting motor output when..."
"There are two types of current limits available: supply and stator." (torque is not really a current limit, it's just peak output settings like peak voltage)